examples/evaluation/Evaluate_RAG_with_LlamaIndex.ipynb

```diff
@@ -0,0 +1,1090 @@
+{
+  "cells": [
+    {
+      "attachments": {},
+      "cell_type": "markdown",
+      "metadata": {},
+      "source": [
+        "# Evaluate RAG with LlamaIndex"
+      ]
+    },
+    {
+      "attachments": {},
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "SxQ2qzb7DPu1"
+      },
+      "source": [
+        "In this notebook we will look into building an RAG pipeline and evaluating it with LlamaIndex. It has following 3 sections.\n",
+        "\n",
+        "1. Understanding Retrieval Augmented Generation (RAG).\n",
+        "2. Building RAG with LlamaIndex.\n",
+        "3. Evaluating RAG with LlamaIndex."
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "jYKmLpi0-AvJ"
+      },
+      "source": [
+        "**Retrieval Augmented Generation (RAG)**\n",
+        "\n",
+        "LLMs are trained on enormous bodies of data but they aren’t trained on your data. Retrieval-Augmented Generation (RAG) solves this problem by adding your data to the data LLMs already have access to. You will see references to RAG frequently in this documentation.\n",
```

Comment:
nit: this almost makes it sound like RAG adds your data to the training data. might be worth rephrasing so this isn't the case